memcached.c

```diff
@@ -3679,6 +3685,8 @@ static int server_socket(const char *interface,
     int error;
     int success = 0;
     int flags =1;
+    socklen_t len;
+    int napi_id;
```

Comment:
doesn't seem to be used anywhere?

I've been moving some of these temp variables to be defined closer to where they're used. ie: in this case both of these are only used inside the `if (settings.num_napi_ids)` block so might as well stuff them in there.

Reply:
ok.